cms/release_calendar/forms.py

```diff
@@ -37,11 +37,10 @@ def clean(self) -> dict:
 
             self.validate_bundle_not_pending_publication(status)
 
-        if status in [ReleaseStatus.CONFIRMED, ReleaseStatus.PUBLISHED]:
-            if not cleaned_data.get("release_date"):
-                raise ValidationError(
-                    {"release_date": "The release date field is required when the release is confirmed"}
-                )
+        if self.instance.status != ReleaseStatus.PROVISIONAL:
+            # Input field is hidden with custom JS for non-provisional releases,
+            # set to None to avoid unexpected behavior
+            cleaned_data["release_date_text"] = None
```

Comment:
```suggestion
            cleaned_data["release_date_text"] = ""
```

Reply:
shall I revert making it nullable?

Reply:
Yes please. `CharField` should not be nullable per se.

Reply:
[12e3f92](https://github.com/ONSdigital/dis-wagtail/pull/158/commits/12e3f9241dd780cbed6f58a67c077b226a07ee51)